src/utilities/theme/legacy-utils.tsx

```diff
@@ -0,0 +1,117 @@
+import tokens from '@shopify/polaris-tokens';
```

Comment:
These are copied over from `utils`, with the only modifications being some type casting and default values to account for the `topBar` theme property going from required to optional.